catalyst/dl/callbacks/checkpoint.py

```diff
@@ -9,19 +9,43 @@
 from catalyst.dl.core import Callback, RunnerState
 
 
-class CheckpointCallback(Callback):
+class BaseCheckpointCallback(Callback):
```

Comment:
I think, `BaseCheckpointCallback` should have:
```
save_checkpoint
remove_checkpoint or truncate_worst_checkpoint ?
get_metrics_file
save_metrics_file
```
?

Reply:
without `get_metrics_file`